mods/voidmulti.sc2mod/base.sc2data/gamedata/abildata.xml

```diff
@@ -236,19 +268,19 @@
         <Flags index="IgnoreUnitCost" value="1"/>
     </CAbilMorph>
     <CAbilMorph id="BurrowZerglingUp">
+        <CmdButtonArray index="Execute">
+            <Flags index="ShowInGlossary" value="0"/>
+        </CmdButtonArray>
+        <Flags index="IgnoreFood" value="1"/>
+        <Flags index="IgnoreUnitCost" value="1"/>
         <InfoArray index="0" RandomDelayMax="0.1125">
             <SectionArray index="Abils">
-                <DurationArray index="Delay" value="0.5625"/>
+                <DurationArray index="Delay" value="0"/>
```

Comment:
**undocumented**: Zergling's Unburrow ability transition delay is now changed from 0.5625 to 0

Does this help in queueing commands like Baneling Morph immdiately?
Burrowed Zergling only has the Unburrow ability.
Other units have 0 in here, so I think this is very likely a good change! I see no reason why Zergling is different from all other units (besides Swarm Host which has a value here as well!)